Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
```

Comment:
Consider using a static version or tag for the Go libraries you are directly using. This prevents potential breaking changes in future releases of the libraries you depend on (e.g., 'github.com/air-verse/air@v1.61.1').